### PR TITLE
One DataModel Ready 3

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/epo/ModelingRules/McafeeEpoModelingRules_1_3/McafeeEpoModelingRules_1_3.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from_xdr_version: 3.5.0
```
Comment (evisochek) [inline Packs/epo/ModelingRules/McafeeEpoModelingRules_1_3/McafeeEpoModelingRules_1_3.yml:1]: @nkanon Please remove this field

File: Packs/ApacheWebServer/ModelingRules/ApacheWebServerModelingRules_1_3/ApacheWebServerModelingRules_1_3_schema.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "Apache_httpd_error": {
+      "_raw_log": {
+        "type": "string",
+        "is_array": false
+      }
+    },
+    "httpd_access": {
```
Comment (evisochek) [inline Packs/ApacheWebServer/ModelingRules/ApacheWebServerModelingRules_1_3/ApacheWebServerModelingRules_1_3_schema.json:8]: @nkanon Why we have 2 datasets in the schema

File: Packs/CheckpointFirewall/ModelingRules/CheckpointFirewall_1_3/CheckpointFirewall_1_3.xif [hunk 3]
```diff
@@ -0,0 +1,177 @@
+[MODEL: dataset ="check_point_vpn_1_firewall_1_raw"]
```
Comment (evisochek) [inline Packs/CheckpointFirewall/ModelingRules/CheckpointFirewall_1_3/CheckpointFirewall_1_3.xif:1]: @nkanon is the "1" should be in the dataset name?

File: Packs/epo/ModelingRules/McafeeEpoModelingRules_1_3/McafeeEpoModelingRules_1_3.yml (hunk 1 above)
Comment (nkanon) [reply Packs/epo/ModelingRules/McafeeEpoModelingRules_1_3/McafeeEpoModelingRules_1_3.yml:1]: Done

File: Packs/CheckpointFirewall/ModelingRules/CheckpointFirewall_1_3/CheckpointFirewall_1_3.xif (hunk 3 above)
Comment (nkanon) [reply Packs/CheckpointFirewall/ModelingRules/CheckpointFirewall_1_3/CheckpointFirewall_1_3.xif:1]: Yes sir, that's how it came directly from the resource. Checked in both SIEM Team and Fusion38.

File: Packs/ApacheWebServer/ModelingRules/ApacheWebServerModelingRules_1_3/ApacheWebServerModelingRules_1_3_schema.json (hunk 2 above)
Comment (nkanon) [reply Packs/ApacheWebServer/ModelingRules/ApacheWebServerModelingRules_1_3/ApacheWebServerModelingRules_1_3_schema.json]: Good catch Sherlock!

**Review by evisochek — APPROVED** [review]

**Pull request merged**